### PR TITLE
[luftdateninfo][sensorcommunity] Rename binding to Sensor.Community

### PR DESCRIPTION
**Pull request by holgerfriedrich**
This would resolve the request to adapt to then new name Sensor.Community which was introduced quite a while ago.
AFAIK the service was renamed and does not use Luftdaten.info any longer.

It would also obsolte PR #14359. which was only to "fix" the documentation.

Of course this PR breaks existing installations, but the name of the service changed.

What is missing:
* add warning note to update.lst (openhab-distro repo) openhab/openhab-distro#1541
* import the German translations properly to Crowdin (the PR just renames the existing _de.properties file, this is not the correct way)
* logo to be updated (-website- docs repo) openhab/openhab-docs#2082
* (optional) create new screenshots for the docs containing English description

@weymann are you still feeling responsible for this binding?
Should we go this way? Otherwise we should close both PRs and leave everything as it is.

### CODE REVIEW

File: bundles/org.openhab.binding.sensorcommunity/README.md
Comment (jlaur) [inline bundles/org.openhab.binding.sensorcommunity/README.md:3]:
```suggestion
Binding for the Sensor Community [Sensor.Community](https://sensor.community/).
The community provides instructions to build sensors on your own and they can be integrated into the database.
```

Do we want to mention here that it was previously known as LuftdatenInfo?

File: bundles/org.openhab.binding.sensorcommunity/src/main/resources/OH-INF/addon/addon.xml
Comment (jlaur) [inline bundles/org.openhab.binding.sensorcommunity/src/main/resources/OH-INF/addon/addon.xml:8]:
```suggestion
	<description>Binding to integrate DIY Sensors from Sensor.Community</description>
```

File: bundles/org.openhab.binding.sensorcommunity/src/main/resources/OH-INF/addon/addon.xml
Comment (jlaur) [inline bundles/org.openhab.binding.sensorcommunity/src/main/resources/OH-INF/addon/addon.xml:7]:
```suggestion
	<name>Sensor.Community Binding</name>
```

I guess?

File: bundles/org.openhab.binding.sensorcommunity/src/main/resources/OH-INF/i18n/sensorcommunity.properties
Comment (jlaur) [inline bundles/org.openhab.binding.sensorcommunity/src/main/resources/OH-INF/i18n/sensorcommunity.properties:3]: Currently this does not correspond to the actual name which is "SensorCommunity Binding", but this file probably will need to be regenerated anyway after changes.

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks for preparing this!

Although it's a completely breaking change, on the other hand it doesn't seem sustainable to keep the current name when it doesn't exist anymore and it's undesired by the community who even requested us to rename it (see https://github.com/openhab/openhab-addons/pull/14359#issuecomment-1419601613).

So I would agree with this change. @openhab/add-ons-maintainers - WDYT?

**Comment by kaikreuzer** [comment]
> So I would agree with this change. https://github.com/orgs/openhab/teams/add-ons-maintainers - WDYT?

I was actually approached at FOSDEM by the project leads of Sensor.Community and they also asked to have it renamed. I understand the request and yes, although it is a completely breaking change, I guess we should follow the request and the 4.0 release sounds like a good time for it.

File: bundles/org.openhab.binding.sensorcommunity/README.md
Comment (holgerfriedrich) [reply bundles/org.openhab.binding.sensorcommunity/README.md:3]: I think we should not mention it. The project wants to get rid if this name.... Anyhow, we will document it in update.lst.

Comment (jlaur) [reply bundles/org.openhab.binding.sensorcommunity/README.md:3]: OK. My thinking was that a search for "openHAB" and "LuftdatenInfo" would then still lead to this page. The project was renamed, and we are now renaming as well to comply, but that doesn't mean it never existed by that name, and we are not erasing history. 🙂 However, you are right that it will also be possible to connect the dots through the release notes.

**Comment by jlaur** [comment]
> * import the German translations properly to Crowdin (the PR just renames the existing _de.properties file, this is not the correct way)

If I read this correctly, the plan is to remove the file from this PR and instead upload it to Crowdin after the PR is merged?

**Comment by holgerfriedrich** [comment]
> > * import the German translations properly to Crowdin (the PR just renames the existing _de.properties file, this is not the correct way)
> 
> If I read this correctly, the plan is to remove the file from this PR and instead upload it to Crowdin after the PR is merged?

This would be the clean way. I think for this PR - which is basically renaming existing stuff - it could be kept in as well. Crowdin will take a few days and a review to complete.....

**Review by jlaur — APPROVED** [review]
LGTM

**Comment by jlaur** [comment]
@holgerfriedrich - I set the "Enhancement" label (even though it's not) to explicitly have it mentioned in the release notes.

**Pull request merged**
